### ISSUE TITLE
3d_vgg_model

### ISSUE THREAD

**Issue by liu666666** [kenshohara/3D-ResNets-PyTorch#17]
@kenshohara Thanks for your wonderful works and sorry for bothering you!
I see that you have released c3d-sports1m-kinetics.t7( 608 MB), did it has the same architecture and get the same performance as the C3D(https://arxiv.org/abs/1412.0767) ? 
Besides, i can not find any results about this model on any dataset in your provided papers. Could you please show the accuarcy on some datasets(ucf101,hmdb51)?
Finally, could you please share the code that finetuning the  c3d-sports1m-kinetics.t7 on the UCF101? 
Thank you in advance!

**Comment by kenshohara** [comment]
c3d-sports1m-kinetics.t7 is the network that has the same architecture as C3D.
But I have not done experiments of C3D on UCF-101 and HMDB-51.
I only fine-tuned the Sports-1M pretrained C3D on Kinetics,
and provided the accuracy of the model on Kinetics.

In addition, I trained the C3D using not PyTorch but Torch.
So, c3d-sports1m-kinetics.t7 cannot be used in this PyTorch code.

If you get UCF-101 or HMDB-51 finetuned C3D model,
* Implement C3D model on PyTorch.
* Convert c3d-sports1m-kinetics.t7 into PyTorch ver.
  * Or convert caffe model of original C3D.
* Fine-tune the model using my code.

I'm sorry but I don't currently have a plan for training C3D model using PyTorch code.

**Comment by liu666666** [comment]
Thanks for your reply! Nice person and wonderful work! @kenshohara 
Could you please share the torch C3D model pretrained only on Sports-1M?

**Comment by kenshohara** [comment]
I used this model (c3d.t7) that is converted from original caffe model of C3D to torch model.
This is the repository for the paper "Long-term temporal convolution for action recognition".
https://github.com/gulvarol/ltc/releases

**Comment by liu666666** [comment]
Thanks for your nice help!
I also have a question, why the size of your provided c3d model(608M) is double size as the official caffe version c3d(300M) and https://github.com/gulvarol/ltc/releases(305M).  How to make it as 300M? Thanks！

**Comment by liu666666** [comment]
The architecture is different from the c3d.  The offical c3d output is 2048, Yours is 4096 and 8192. Therefore, your provided model is larger.

**Comment by kenshohara** [comment]
I have not considered the difference of sizes...
I confirmed the architecture of our c3d-sports1m-kinetics.t7 and gulvarol's c3d.t7 again
but the architecture is not different.
I'm not sure why the size is different.

**Comment by liu666666** [comment]
Thanks for your reply! 
This line (https://github.com/kenshohara/3D-ResNets/blob/1acafae56406c8c518b8825cc34bd70ae1de1a2d/models/resnet.lua#L120) causes the model size larger(600M), could you please give me some ideas to make it the same as the official model(300M)
